inc/namespace.php

```diff
@@ -246,6 +262,130 @@ function noop_wp_query_found_rows_on_failed_ep_request( string $sql, WP_Query $q
 	return '';
 }
 
+/**
+ * Add default initial options and settings on install.
+ *
+ * @return void
+ */
+function on_wp_install() {
+	// This option is used to determine the index name for backwards compat.
+	set_index_version( 1 );
```

Comment:
Should this be `3` instead? Thinking it might be good to match the EP version itself.

Reply:
Hmm, yeah I take your point, though I was thinking of using this value for the zero downtime reindexing.